ReactSkia/components/RSkComponentImage.h

```diff
@@ -1,16 +1,25 @@
 #pragma once
 
+#include "include/core/SkRect.h"
+
+#include "react/renderer/mounting/ShadowView.h"
```

Comment:
ShadowView.h is already included by RSkComponent.h .
So this include is not required here.

Reply:
Not required.